src/Internals/SectionHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from "../utils/styled"
+
+export const SectionHeader = styled("div")(({ theme }) => ({
+  fontFamily: theme.font.family.main,
+  fontSize: theme.font.size.body,
+  fontWeight: theme.font.weight.medium,
+  display: "flex",
+  alignItems: "center",
+  backgroundColor: theme.color.background.lighter,
+  color: theme.color.text.dark,
+  flex: "0 0 auto", // Make sure it stays the same size if other parts of the card push it
```

Comment:
We can fix this now 😄 

```suggestion
  flex: "0 0", // Make sure it stays the same size if other parts of the card push it
```